api/apps/api/src/modules/scenario-specification/application/activate-candidate-specification.command.ts

```diff
@@ -0,0 +1,26 @@
+import { Command } from '@nestjs-architects/typed-cqrs';
+import { Either } from 'fp-ts/Either';
+import {
+  NoCandidateToActivate,
+  SpecificationIsNoLongerACandidate,
+} from '../domain';
+
+export const ScenarioSpecificationNotFound = Symbol(
```

Comment:
why symbols are started with a capital letter? I'd expect them to be a class or a type then, but It's a value

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/416/commits/4728b10a6bddf2263ed43523b720cc325d6ae2dc